api-server/src/server/utils/redirection.js

```diff
@@ -70,14 +67,14 @@ function getRedirectParams(req, _normalizeParams = normalizeParams) {
   return _normalizeParams({ returnTo: returnUrl.href, origin, pathPrefix });
 }
 
-function isRootPath(redirectBase, returnUrl) {
+function haveSamePath(redirectBase, returnUrl) {
   const base = new URL(redirectBase);
   const url = new URL(returnUrl);
   return base.pathname === url.pathname;
 }
 
 module.exports.getReturnTo = getReturnTo;
-module.exports.getRedirectBase = getRedirectBase;
```

Comment:
A few of the other tests still rely on this export I think.

Reply:
Sorry, Nick, I rushed this and somehow evaded the linter in the process.